portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase

```diff
@@ -40,4 +103,85 @@ definition {
 		ACUtils.assertXAxisTimedGraph(parentName = "Beryl Commerce Activities");
 	}
 
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8280 | Test Summary: Anon and known visitors legend shown"
+	@ignore = "true"
+	@priority = "5"
+	test CanShownAnonAndKnownVisitorsLegend {
+
+		// TODO LRAC-8280 CanShownAnonAndKnownVisitorsLegend
+
+	}
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8281 | Test Summary: Anon and known visitors popover shows breakdown"
+	@ignore = "true"
+	@priority = "5"
+	test CanShownAnonAndKnownVisitorsPopoverBreakdown {
+
+		// TODO LRAC-8281 CanShownAnonAndKnownVisitorsPopoverBreakdown
+
+	}
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8295 | Test Summary: Assert Sites Activity time filter able to use Custom Range"
+	@ignore = "true"
+	@priority = "5"
+	test CanSitesActivityTimeFilterUseCustomRange {
+
+		// TODO LRAC-8295 CanSitesActivityTimeFilterUseCustomRange
+
+	}
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8290 | Test Summary: Unique Visitors label is shown on the tabs of Pages dashboards"
+	@ignore = "true"
+	@priority = "5"
+	test CanUniqueVisitorsLabelShowPagesDashboards {
+
+		// TODO LRAC-8290 CanUniqueVisitorsLabelShowSitesDashboards
```

Comment:
hi @rafaellajordao it should be this // TODO LRAC-8290 CanUniqueVisitorsLabelShowPagesDashboards